### PR TITLE
Merging form-block with aem-boilerplate-forms

### PR DESCRIPTION
**Pull request by neerajadobe93**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
https://main--aem-boilerplate-forms--adobe-rnd.hlx.page/

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -17,5 +17,6 @@ module.exports = {
     'import/extensions': ['error', {
       js: 'always',
     }],
+    'no-console': 'off',
```
Comment (vdua) [inline .eslintrc.cjs]: keep it on.

**Review by vdua — APPROVED** [review]

File: .github/workflows/cleanup-on-create.yaml [hunk 2]
```diff
@@ -27,6 +28,21 @@ jobs:
...
+
+      - name: Update .eslintignore
+        run: |
+          sed -i '/^rollup/d' .eslintignore
+          sed -i '/^helix-importer-ui/d' .eslintignore 
+
+      - name: Update Package.json
+        run: |
+          rm -rf package.json
+          mv package-prod.json package.json
```
Comment (vdua) [inline .github/workflows/cleanup-on-create.yaml:44]: we have to either remove package-lock or update it

**Review by vdua — CHANGES_REQUESTED** [review]
Minor comments

**Review by jalagari — APPROVED** [review]

File: .github/workflows/cleanup-on-create.yaml (hunk 2 above)
Comment (gargadobe) [reply .github/workflows/cleanup-on-create.yaml:44]: shall i remove  the package-lock.json from repo itself ?

Comment (gargadobe) [reply .github/workflows/cleanup-on-create.yaml:44]: Removed the package-lock.json from repo

Comment (vdua) [reply .github/workflows/cleanup-on-create.yaml:44]: I think I did not communicate properly. We have to remove the file when creating the template. During development we would need to check-in that file.

File: .eslintrc.cjs [hunk 3]
```diff
@@ -17,5 +17,6 @@ module.exports = {
     'import/extensions': ['error', {
       js: 'always',
     }],
+    'no-console': 'on',
```
Comment (vdua) [inline .eslintrc.cjs:20]: remove this. I think the default is on only

**Review by vdua — APPROVED** [review]

**Pull request merged**